### DISCUSSION TITLE
Durata OTP

### DISCUSSION THREAD

**Discussion by rvinzani** [italia/ansc#768] [Miglioramenti] [answered]
Lissone

Sarebbe possibile incrementare la validità temporale dell’OTP da 4 a 6 ore?

**Comment by mttfranci** [comment, marked as answer]
Siamo consapevoli che la richiesta è stata reiterata più volte, ma stiamo ancora valutando le implicazione dal punto di vista della sicurezza, pertanto non riteniamo di apportare modifiche al comportamento almeno fino al termine dell'adozione controllata.

Saluti

Assistenza ANSC
